alibi_detect/cd/classifier.py

```diff
@@ -16,7 +20,7 @@ class ClassifierDrift:
     def __init__(
             self,
             x_ref: Union[np.ndarray, list],
-            model: Callable,
+            model: Union[ClassifierMixin, Callable],
```

Comment:
Not really related to this PR, but we should type `model` with the `tensorflow` and `torch` types not general `Callable`. This would require `if TYPE_CHECKING: import tensorflow, torch` at the top of the module.